.ci/39.yaml

```diff
@@ -2,24 +2,24 @@ name: test
 channels:
   - conda-forge
 dependencies:
-  - python =3.9
-  - scipy >=1.0
-  - numpy >=1.3
-  - pandas >=1
-  - networkx
-  - libpysal
+  - python=3.9
   - geopandas>=0.7.0
+  - libpysal
   - matplotlib
+  - networkx
+  - numpy>=1.3
+  - pandas>=1
+  - pulp
   - scikit-learn>=0.22
+  - scipy>=1.0
   - spenc
+  # testing, docs, etc.
+  - codecov
+  - coverage
+  - nbsphinx
+  - numpydoc
   - pytest
   - pytest-cov
-  - pulp
-  - coverage
-  - coveralls
   - sphinx>=1.4.3
   - sphinxcontrib-bibtex==1.0.0
   - sphinx_bootstrap_theme
-  - numpydoc
-  - nbsphinx
```

Comment:
Don't we need to leave all the doc stuff in here since this environments used to build the docs in [`build_docs.yml`](https://github.com/pysal/spopt/blob/1609f377226a2d95a2cc24026ba355e9b0fdb682/.github/workflows/build_docs.yml#L16).

Reply:
yeah, the yamls in `.ci/` need to be overly complete (`requirements_[all]`), the `environment.yml` should mirror `requirements.txt`

i think its just a formatting thing on the diff though... they're still in there, they just got moved up

Reply:
> i think its just a formatting thing on the diff though... they're still in there, they just got moved up

Ah, correct. That was my oversight.